portal/src/components/Collections/Collection/Collection.component.js

```diff
@@ -54,6 +55,28 @@ export default {
           ? PublishStatus.PUBLISHED
           : PublishStatus.DRAFT
       }
+    },
+    communityTitle() {
+      if (!this.collection || !this.collection.communities[0]) {
+        return
+      }
+      const communityDetails = this.collection.communities[0].community_details.find(
+        details => {
+          return details.language_code.toLowerCase() === this.$i18n.locale
+        }
+      )
+      return communityDetails.title
+    },
+    communityLink() {
+      if (!this.collection || !this.collection.communities[0]) {
+        return
+      }
+      const path = localePath({
```

Comment:
is dit niet genoeg voor een relative path?

Reply:
Nee, het geeft een object terug, waar alleen ___<locale> is toegevoegd aan de name van de input config.